docs/user/installation_and_configuration.rst

```diff
@@ -658,6 +658,21 @@ After building, extract ``target/geomesa-cassandra-gs-plugin_2.11-$VERSION-insta
 
 Restart GeoServer after the JARs are installed.
 
+Additional Configuration
+------------------------
+
+GeoMesa uses a site xml file to maintain system property configurations. This file can be found
+at ``conf/geomesa-site.xml`` of either the Accumulo or Kafka distributions. The default settings
```

Comment:
is this meant to be the geomesa accumulo distribution, or the actual accumulo install? seems like it should be the geomesa one right?